featurebook/05 - Email notification settings.md

```diff
@@ -0,0 +1,35 @@
+# Featurebook > 05 - Email notification settings.md
+
+Go to [Featurebook > Index](../FEATUREBOOK.md)
+
+`@Scenario:`
+
+## WHEN a notification email is sent and the "Teaser" option is selected in the plugin settings, ...
+
+* THEN the email contains only a short teaser text (100 characters) instead of the full note
+
+`@Scenario:`
+
+## WHEN a notification email is sent and the "Whole note" option is selected in the plugin settings, ...
+
+* THEN the email contains the full content of the note
+
+![notification mail content settings](img/notificationMailContentSetting.png)
+![notification mail content options](img/notificationMailContentOptions.png)
+
+
+`@Scenario:`
+
+## GIVEN there is at least one role selected in the multi-select list box in the plugin settings, ...
```

Comment:
Ca mai sus ar fi bine sa pui textul setarii in ghilimele in loc de "multi-select ..."

Reply:
Done